protoletariat/fdsetgen.py

```diff
@@ -14,6 +14,11 @@
 
 _PROTO_SUFFIX_PATTERN = re.compile(r"^(.+)\.proto$")
 
+def _utf8(x) -> str:
```

Comment:
Can you remove this function and instead replace the calls with `os.fsdecode`? https://docs.python.org/3/library/os.html#os.fsdecode